assets/app/game_manager.rb

```diff
@@ -54,6 +54,15 @@ def get_games(params = nil)
     end
   end
 
+  def get_game(id = nil)
+    id ||= @game_data['id']
+
+    @connection.safe_get("/game/#{id}") do |data|
+      data[:loaded] = @game_data[:loaded] if @game_data
```

Comment:
i forgot how this code works, can you explain what this is doing?

Reply:
The api returns the object with loaded=true (I don't know why). When the game starts the page automatically enters the game (same url), but if I left loaded=true it doesn't create everything that needs to be created.

Reply:
ok i understand, i think we can clean this up a bit,

when you hit /game/:id, this is the full game with all actions which is not needed for the game invite,

what you need is an API to get the game metadata only, so perhaps you can do /game/:id?meta_only=1 to not load the actions and then loaded would not be true

Reply:
This is only for the invitation page, so the actions will be empty (or at worst they will be very few only once)